src/sentry/migrations/0888_groupopenperiod_gist_index_in_code.py

```diff
@@ -0,0 +1,52 @@
+# Generated by Django 5.2.1 on 2025-05-13 19:59
+
+import django.contrib.postgres.constraints
+import django.contrib.postgres.fields.ranges
+from django.db import migrations, models
+
+import sentry.models.groupopenperiod
+from sentry.new_migrations.migrations import CheckedMigration
+
+
+class Migration(CheckedMigration):
+    # This flag is used to mark that a migration shouldn't be automatically run in production.
+    # This should only be used for operations where it's safe to run the migration after your
+    # code has deployed. So this should not be used for most operations that alter the schema
+    # of a table.
+    # Here are some things that make sense to mark as post deployment:
+    # - Large data migrations. Typically we want these to be run manually so that they can be
+    #   monitored and not block the deploy for a long period of time while they run.
+    # - Adding indexes to large tables. Since this can take a long time, we'd generally prefer to
+    #   run this outside deployments so that we don't block them. Note that while adding an index
+    #   is a schema change, it's completely safe to run the operation after the code has deployed.
+    # Once deployed, run these manually via: https://develop.sentry.dev/database-migrations/#migration-deployment
+
+    is_post_deployment = False
+
+    dependencies = [
+        ("sentry", "0887_environment_deleted_column"),
+    ]
+
+    operations = [
+        migrations.SeparateDatabaseAndState(
+            state_operations=[
```

Comment:
Might only be working because this is state only

Reply:
Weird, I thought our problem was needing `gist_int8_ops` as part of the constraint, but I guess we just needed that extension instead.